js/src/kb-search/wp-seo-kb-search-init.js

```diff
@@ -0,0 +1,55 @@
+/* global wpseoAdminL10n */
+
+import React from "react";
+import ReactDom from "react-dom";
+import AlgoliaSearcher from "./wp-seo-kb-search.js";
+
+var intialiseAlgoliaSearch = () => {
```

Comment:
I think it does pretty much describe what is done here. What do you think is a better name then?